backend/benefit/applications/api/v1/views.py

```diff
@@ -154,7 +154,23 @@ def simplified_application_list(self, request):
         )
 
         serializer = self.serializer_class(qs, many=True, context=context)
-        return Response(serializer.data, status=status.HTTP_200_OK)
+
+        order_by = request.GET.get("order_by")
+        is_order_descending = True if order_by and order_by.lower() == "desc" else False
```

Comment:
You can use `bool(order_by and order_by.lower() == "desc")`.

Reply:
Also, here usage of `.get()`'s default value could maybe simplify some things :)